### ISSUE TITLE
What is Deploy Network?

### ISSUE THREAD

**Issue by Anish-Agnihotri** [deploynetwork/Overview#1]
Deploy Network is a simple web platform that allows individuals to spin up synced P2P networking protocols (including, but not limited to: Ethereum, Bitcoin, Swarm, and Status nodes) in just a few minutes. It utilities [DAppNode](https://dappnode.io) custom ISO's on bleeding-edge server architecture to offer powerful instances, and makes use of the [ERC-1337](https://1337alliance.com/) subscriptions standard on the Ethereum network to handle recurring server subscriptions.

In other words, it allows users to deploy their own blockchain nodes with one-click, without having the hassle of managing server architecture, maintaining their nodes, and troubleshooting complicated protocols - all at an affordable price.
